### PR TITLE
Create about-2 pattern

### PR DESCRIPTION
**Pull request by richtabor**
We can adjust the nomenclature and translations later. 

CleanShot 2023-09-05 at 16 29 06
CleanShot 2023-09-05 at 16 29 23

### CODE REVIEW

**Comment by richtabor** [comment]
One of the patterns in https://github.com/WordPress/twentytwentyfour/issues/190.

File: patterns/about-2.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Title: About 2
+ * Slug: twentytwentyfour/about
+ * Categories: about, featured, text
```
Comment (MaggieCabrera) [inline patterns/about-2.php]: I agree we can decide on the naming later but now we have 2 patterns with the same slug. Also, about is not a default category, is it?

**Review by MaggieCabrera — APPROVED** [review]
Just one comment, with that change this is good to go

File: patterns/about-2.php (hunk 1 above)
Comment (richtabor) [reply patterns/about-2.php]: I'll change the slug to match the name (about-2). About is a default category; although we don't use it much (yet).

Comment (MaggieCabrera) [reply patterns/about-2.php]: TIL, thank you!

**Pull request merged**